conn.go

```diff
@@ -609,9 +609,11 @@ func (c *conn) connWriter() {
 		// connection complete
 		case <-c.done:
 			// send close
+			close := &performClose{}
```

Comment:
Can you rename this so it doesn't shadow the `close` built in function?

Reply:
Thanks for the review, change made.